src/ExtranetBundle/objects/WebModules/MTExtranetCore/MTExtranetCoreEndPoint.class.php

```diff
@@ -220,9 +220,7 @@ public function Register($sSuccessURL = null, $sFailureURL = null)
                 }
             }
 
-            if ($bDataValid) {
-                $oUser->Register();
-
+            if ($bDataValid && true === $oUser->Register()) {
```

Comment:
I'd make two lines of this:

`
if (true === $bDataValid) {
    $bDataValid = $oUser->Register();
}

if (true === $bDataValid) {
....

`

Reply:
ok